release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -161,6 +161,41 @@ The documentation for {olmv1}, which has been in Technology Preview starting in
 
 The updated location and guide name reflect a more focused documentation experience and aims to differentiate between {olmv1} and {olmv0}.
 
+[id="ocp-4-17-extensions-new-features_{context}"]
+==== {olmv1} Technology Preview features
+
+This release introduces the following new features to {olmv1} Technology Preview:
+
+Custom resource definition (CRD) upgrade safety::
+When you update a custom resource definition (CRD) that is provided by a cluster extension, {olmv1-first} now runs a CRD upgrade safety preflight check to ensure backwards compatibility with previous versions of that CRD. The CRD update must pass the validation checks before the change is allowed to progress on a cluster.
++
+For more information, see xref:../extensions/ce/crd-upgrade-safety.adoc#crd-upgrade-safety[Custom resource definition (CRD) upgrade safety].
+
+Single object ownership for cluster extensions::
+In {olmv1}, a Kubernetes object can only be owned by a single `ClusterExtension` object at a time. This ensures that objects within an {product-title} cluster are managed consistently and prevents conflicts between multiple cluster extensions attempting to control the same object.
++
+For more information, see _Object ownership for cluster extensions_.
+
+Enhanced security::
+{olmv1} now requires a dedicated service account for installing, updating, and managing cluster extensions. Additionally, catalogd uses HTTPS encryption to secure catalog server responses.
++
+For more information, see _Creating a service account to manage cluster extensions_.
```

Comment:
Seems like this `_Creating` should be a link.

Reply:
@jianzhangbjz As soon as https://github.com/openshift/openshift-docs/pull/82245 merges, I can create the cross reference link.

Reply:
Addressed. Since @jianzhangbjz is on vacation, can another member of the OLM QE team, PTAL?

cc: @kuiwang02, @Xia-Zhao-rh, @bandrade

Reply:
/lgtm